apps/courses/templates/courses/cms/_course_detail.scss

```diff
@@ -0,0 +1,60 @@
+// Overridable & namespaced global variables
+$course-detail-foo: 15rem !default;
```

Comment:
I guess this is a copy/pasta left over? I don't see it reused anywhere

Reply:
Yep, exactly.